src/app/modal-gallery/modal-gallery.component.ts

```diff
@@ -512,7 +520,8 @@ export class ModalGalleryExampleComponent implements OnDestroy {
       id,
       images: imagesArrayToUse,
       currentImage: imageToShow,
-      libConfig
+      libConfig,
+      previewsTemplate: this.previewsTemplate,
```

Comment:
previewsTemplate should be applied only to a specific example, otherwise all examples in this demo app will use that template

Reply:
I suggest, for instance to add a new exaperimental example at the end on the page with a dedicare "openModalWithPreviewsTemplate" or something similar

Reply:
Sure, I set it quickly here for the sake of the discussion. It'll have its own example when I finalize the PR.